### ISSUE TITLE
'adb devices' exited with code null

### ISSUE THREAD

**Issue by appiumnovice** [appium/appium#8437]
## The problem

I have a testset of hundreds tests.
Regularly, the install of the app is blocked and Appium wait for 20 minutes to put the test fail and to pass to the next test (which fails too in the same conditions !).

## Environment

* Appium version (or git revision) that exhibits the issue: 1.6.5-beta
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium: Mac OSX El Capitan 10.12.4
* Node.js version (unless using Appium.app|exe):
* Mobile platform/version under test: Android 4 -> 7
* Real device or emulator/simulator: Android Studio Emulators
* Appium CLI

## Details

When I try to stop Appium server with Ctrl C, it unblocks the install of the app and Appium display the error "Error while getting connected devices. Original error: Command '/Users/stephane_ducornet/Library/android/sdk/platform-tools/adb devices' exited with code null"
Appium continues to install app, with success.

## Link to Appium logs

https://gist.github.com/appiumnovice/12f947425fe83e8c1d1e70e4896a9017

**Comment by mykola-mokhnach** [comment]
Closed as duplicate of https://github.com/appium/appium/issues/8312
